ReactNativeClient/lib/shim-init-node.js

```diff
@@ -412,6 +419,24 @@ function shimInit() {
 	shim.pathRelativeToCwd = (path) => {
 		return toRelative(process.cwd(), path);
 	};
+
+	shim.isKeytarAvailable = () => {
+		return shim.isElectron() && !shim.isPortable() && keytar;
+	};
+
+	shim.loadSecureItems = (appId) => {
+		if (shim.isKeytarAvailable()) {
+			return keytar.findCredentials(appId);
+		}
+		return Promise.resolve([]);
+	};
+
+	shim.saveSecureItem = (appId, item) => {
+		if (shim.isKeytarAvailable()) {
+			return keytar.setPassword(appId, item.key, item.value);
+		}
+		return false;
+	};
```

Comment:
Please make these methods async. Keytar might be synchronous, but it's likely that for example a React Native lib would be async, so it needs to work with both.